app/views/director/counselors/new.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1> Invite a counselor! </h1>
+
+<%= form_for @counselor, url: "/counselors" do |form| %>
```

Comment:
Have you verified this works? Previously you were using `<model>_path` - is this different than that convention?